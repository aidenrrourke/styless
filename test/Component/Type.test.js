import React from 'react';
import renderer from 'react-test-renderer';
import styled from 'styled-components';
import 'jest-styled-components';

test('Returns true if a value is a number, false otherwise.', () => {
    const Div = styled.div`
        number1: isnumber(#ff0);     // false
        number2: isnumber(blue);     // false
        number3: isnumber("string"); // false
        number4: isnumber(1234);     // true
        number5: isnumber(56px);     // true
        number6: isnumber(7.8%);     // true
        number7: isnumber(keyword);  // false
        number8: isnumber(url("http://yadi")); // false
	`;
    expect(renderer.create(<Div/>).toJSON()).toMatchSnapshot();
});