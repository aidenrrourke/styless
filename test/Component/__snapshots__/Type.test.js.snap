// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Returns true if a value is a number, false otherwise. 1`] = `
.c0 {
  number1: false;
  number2: false;
  number3: false;
  number4: true;
  number5: true;
  number6: true;
  number7: false;
  number8: false;
}

<div
  className="c0"
/>
`;
